Drop-In/src/main/java/com/braintreepayments/api/DropInClient.java

```diff
@@ -486,6 +488,33 @@ public void launchDropInForResult(FragmentActivity activity, int requestCode) {
         });
     }
 
+    /**
+     * Called to launch a {@link DropInActivity}.
+     *
+     * NOTE: This method requires {@link DropInClient} to be instantiated with either an Activity
+     * or with a Fragment.
+     *
+     * @see #DropInClient(Fragment, DropInRequest, String)
+     * @see #DropInClient(Fragment, DropInRequest, ClientTokenProvider)
+     * @see #DropInClient(FragmentActivity, DropInRequest, String)
+     * @see #DropInClient(FragmentActivity, DropInRequest, ClientTokenProvider)
+     */
+    public void launchDropIn() {
+        getAuthorization((authorization, authorizationError) -> {
+            if (authorization != null) {
+                if (observer != null) {
```

Comment:
Is there a way to combine these nested if's into an `&&` to just single line and simplify it a bit?

Reply:
Yeah definitely that's a good call. I'll simplify the nesting structure here.